docs/source/x_ext.rst

```diff
@@ -947,6 +990,7 @@ The following handshake pairs exist on the eXtension interface:
 
 * ``compressed_valid`` with ``compressed_ready``.
 * ``issue_valid`` with ``issue_ready``.
+* ``register_valid`` with implicit always ready signal.
```

Comment:
No accurate for scenario 1

Reply:
i think the best would be to add the ready signal. An note it can be wired to 1 in the second scenario. The same was done for the rs_valid signals, so then this is aligned.

Reply:
It now states 'register_valid with issue_ready' (should use register_ready instead)

Reply:
fixed in f9b96bb3563acc2f46d8058afd5b5e46ba20b1c3